.teamcity/settings.kts

```diff
@@ -422,4 +424,89 @@ object NightlyTests : BuildType({
             }
         }
     }
-})
\ No newline at end of file
+})
+
+object Pixi : Project({
+    name = "Pixi"
+
+    buildType(UpdateDependencies)
+})
+
+object UpdateDependencies : BuildType({
+    name = "Update Dependencies"
+
+    params {
+        param("GT_USER", "deltares-service-account")
+        password("env.GH_TOKEN", "credentialsJSON:ced4991a-a11d-463b-b79a-a657eea89523")
+    }
+
+    vcs {
+        root(AbsoluteId("iMOD6_IMODPython_ImodPython"), "+:. => imod-python")
+
+        cleanCheckout = true
+    }
+
+    steps {
+        powerShell {
+            name = "Download Github CLI"
+            id = "Download_Github_CLI"
+            formatStderrAsError = true
+            scriptMode = script {
+                content = """
+                    Invoke-WebRequest -Uri  https://github.com/cli/cli/releases/download/v2.42.1/gh_2.42.1_windows_amd64.zip -OutFile gh.zip
+                    Expand-Archive gh.zip -DestinationPath gh
```

Comment:
You can just add `gh` as pixi dependency.
Alternative it's also available as module on linux called `github`.
I would suggest the former though.

Reply:
I'll be going for your first suggestion. Mainly because I don't want to introduce a new scripting language besides the ones that already being used